docs/release-notes/release-notes-0.19.0.md

```diff
@@ -196,4 +199,4 @@ The underlying functionality between those two options remain the same.
 * Oliver Gugger
 * Pins
 * Viktor Tigerström
-* Ziggie
\ No newline at end of file
+* Ziggie
```

Comment:
while we are here, can we add a new line pls? this keeps getting changed back and forth on various PRs 😂

Reply:
haha sure